TradingOnTalentURAP/degree_labelling.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd
+
+# Labelling Function
+
+def assign_labels(df, direct_keywords, direct_star_keywords, exclude_keywords, tangential_keywords):
```

Comment:
Please add detailed comments, since the dataframe syntax really obfuscates what's going on. It's not that easy to read in my opinion... I hope it's at least computationally more efficient...